docs/source/release.rst

```diff
@@ -142,6 +126,23 @@ In case the action for the automatic release on PyPI fails, the commands to rele
     You need to be a registered maintainer of Mitiq project on PyPI to upload
     a new release on PyPI from your local machine.
 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Start the new version on Github
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. note::
+    You need to have write access to the Mitiq Github repository to make
+    a new release.
+
+------------------------------------------------
+Update the new development version
+------------------------------------------------
+
+Add a new section to the ``CHANGELOG.md`` to track changes in the following
+release, meaning that if ``vX.Y.Z`` was just released, then there should be
+a section for ``vX.(Y+1).Z`` that is marked "In Development". Also, change the
+version in the ``VERSION.txt`` file from ``vX.Y.Z`` to ``vX.(Y+1).Zdev``.
```

Comment:
```suggestion
a section for ``vX.(Y+1).0`` that is marked "In Development". Also, change the
version in the ``VERSION.txt`` file from ``vX.Y.Z`` to ``vX.(Y+1).0dev``.
```
I think the third number should go to 0. If I am wrong, please ignore this suggestion.

Reply:
You are def right! fixing in next PR